merlin/models/tf/inputs/embedding.py

```diff
@@ -150,6 +150,182 @@ def from_config(cls, config):
 
         return cls(dim, *schema, **config)
 
+@tf.keras.utils.register_keras_serializable(package="merlin.models")
```

Comment:
Might be better to move this to another file, or only include this class when the user has the sok installed. This because we don't want the sok to be a required dependency.

Reply:
> OK, any other comments

Reply:
Do you have any suggestions?

Reply:
Moved to a separate file in [7555c6f](https://github.com/NVIDIA-Merlin/models/pull/863/commits/7555c6f8fbc42db7def23309b757a6c9e7fc7208).